sample/src/androidTest/java/com/schibsted/spain/barista/sample/LongClickTest.java

```diff
@@ -0,0 +1,61 @@
+package com.schibsted.spain.barista.sample;
+
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static com.schibsted.spain.barista.BaristaAssertions.assertDisplayed;
+import static com.schibsted.spain.barista.BaristaClickActions.longClick;
+
+@RunWith(AndroidJUnit4.class)
+public class LongClickTest {
+
+  @Rule
+  public ActivityTestRule<FlowFirstScreen> activityRule = new ActivityTestRule<>(FlowFirstScreen.class);
+
+  @Test
+  public void checkLongClick_byId() {
+    longClick(R.id.next);
+    assertDisplayed("I was long pressed");
+  }
+
+  @Test
+  public void checkLongClick_byText() {
+    longClick("Next");
+    assertDisplayed("I was long pressed");
+  }
+
+  @Test
+  public void checkClick_byStringResource() {
```

Comment:
I think you renamed the two first tests with "LongClick" but forgot to update the rest. Could you change them so they're not duplicate with the normal click tests?